plugin/js/content.js

```diff
@@ -130,6 +136,53 @@ function fireEvent(event) {
   }
 }
 
+function isNetflix() {
+  // are we on the netflix.com page?
+  return location.host === "www.netflix.com";
+}
+
+function fireEventNetflix(event) {
+  recieved = true;
+  recievedEvent = event.type;
```

Comment:
It's already in `fireEvent` function. Why not place it on top of  `fireEvent` and remove it from there.